ios/FluentUI/Button/ButtonTokenSet.swift

```diff
@@ -15,6 +15,17 @@ public enum ButtonStyle: Int, CaseIterable {
     case danger
     case dangerOutline
     case dangerSubtle
+    case floatingAccent
+    case floatingSubtle
+
+    public var isFab: Bool {
```

Comment:
nit: Could we changes this to `isFloating`? I see in the tokens branch we had `isFloatingStyle`, but I feel like `button.style.isFloatingStyle` has too many styles. But I'm not sure everyone will know that "fab" means "Floating Action Button"